protos/flyteidl/admin/common.proto

```diff
@@ -210,9 +210,13 @@ message ResourceListRequest {
     // +optional
     string filters = 4;
 
-    // Sort ordering.
+    // Sort column ordering.
     // +optional
-    Sort sort_by = 5;
+    // Deprecated. Use sort_keys instead.
+    Sort sort_by = 5 [deprecated = true];
+
+    // Sort column(s) with direction.
+    repeated Sort sort_keys = 6;
```

Comment:
does this work with http requests in grpc gateway v1? 
I recall we used unstructured string for filters (https://docs.flyte.org/en/latest/concepts/admin.html#adding-request-filters)

>  While this allows for a single implementation for both HTTP and gRPC, an important limitation is that fields mapped to the path pattern cannot be repeated and must have a primitive (non-message) type. Unfortunately this means that repeated string filters cannot use a proper protobuf message.